.github/workflows/ingestion-deploy-dev.yaml

```diff
@@ -1,4 +1,4 @@
-name: Deploy ECS to Dev
+name: Deploy Ingestion to Dev
```

Comment:
better name. this is what we see on the github actions tab so it needs to be good.